Assets/MRTK/SDK/Experimental/Elastic/Scripts/ElasticTypes.cs

```diff
@@ -9,24 +9,21 @@
 
 namespace Microsoft.MixedReality.Toolkit.Experimental.Physics
 {
-
     /// <summary>
-    /// Properties of the extent in which a damped
-    /// harmonic oscillator is free to move.
+    /// Properties of a linear, one-dimensional extent
+    /// in which a damped harmonic oscillator is free to move.
     /// </summary>
     [Serializable]
-    public struct ElasticExtentProperties<T>
+    public struct LinearElasticExtent
```

Comment:
Generically typed extent structs are no more! They are replaced with specific structs used by each type of elastic system. Unfortunately, generically typed structs are **not serializable** in Unity 2018 or 2019. Unity 2020.1 supports generic serialization at a certain level, but it is not widely supported.